iocore/net/SSLConfig.cc

```diff
@@ -112,7 +112,7 @@ SSLConfigParams::cleanup()
   ats_free_null(client_cipherSuite);
   ats_free_null(dhparamsFile);
   ats_free_null(ssl_wire_trace_ip);
-  ats_free_null(ticket_key_filename);
+  ticket_key_filename = (char *)ats_free_null(ticket_key_filename);
```

Comment:
Let's go bigger. Pull the code out of the constructor, make it a method, and then call that at the end of this method. Reset everything, really clean up.